public/components/trace_analytics/components/traces/traces_custom_indices_table.tsx

```diff
@@ -47,179 +23,111 @@ interface TracesLandingTableProps {
-    </EuiSmallButtonEmpty>
-  );
+  const renderCellValue = useMemo(() => {
+    return ({ rowIndex, columnId }: { rowIndex: number; columnId: string }) => {
+      const isTracesMode = props.tracesTableMode === 'traces';
+      const adjustedRowIndex = isTracesMode
+        ? rowIndex
+        : rowIndex - pagination.pageIndex * pagination.pageSize;
 
-  const renderTableOptions = (option: EuiSelectableOption, searchValue: string) => {
-    return (
-      <>
-        <EuiHighlight search={searchValue}>{TRACE_TABLE_TITLES[option.key]}</EuiHighlight>
-        <br />
-        <EuiTextColor color="subdued" className="popOverSelectableItem">
-          <small>{option['aria-describedby']}</small>
-        </EuiTextColor>
-      </>
-    );
-  };
+      if (!items.hasOwnProperty(adjustedRowIndex)) return '-';
+      const value = items[adjustedRowIndex]?.[columnId];
 
-  const renderTitleBar = (totalItems?: number) => {
-    return (
-      <EuiFlexGroup justifyContent="spaceBetween" gutterSize="s">
-        <EuiFlexItem grow={false}>
-          <EuiPopover
-            button={titlePopoverButton(totalItems)}
-            isOpen={isTitlePopoverOpen}
-            closePopover={() => setIsTitlePopoverOpen(false)}
-          >
-            <EuiPopoverTitle className="tableModePopover">
-              Select trace table filter
-            </EuiPopoverTitle>
-            <EuiSelectable
-              singleSelection="always"
-              aria-label="Searchable trace mode"
-              searchable
-              searchProps={{
-                'data-test-subj': 'traceTableMode',
-              }}
-              renderOption={renderTableOptions}
-              listProps={{ rowHeight: 80 }}
-              options={tableOptions}
-              onChange={(newOptions) => {
-                setTableOptions(newOptions);
-                const tableMode = newOptions.filter((option) => option.checked === 'on')[0]
-                  .key as TraceQueryMode;
-                props.setTracesTableMode(tableMode);
-                sessionStorage.setItem(TRACE_TABLE_TYPE_KEY, tableMode);
-                setIsTitlePopoverOpen(false);
-              }}
-            >
-              {(list, search) => (
-                <Fragment>
-                  {search}
-                  {list}
-                </Fragment>
-              )}
-            </EuiSelectable>
-          </EuiPopover>
-        </EuiFlexItem>
-        {props.tracesTableMode !== 'traces' && (
-          <EuiFlexItem grow={false}>
-            <EuiSmallButtonEmpty onClick={() => setShowAttributes(!showAttributes)}>
-              {showAttributes ? 'Hide Attributes' : 'Show Attributes'}
-            </EuiSmallButtonEmpty>
-          </EuiFlexItem>
-        )}
-      </EuiFlexGroup>
-    );
-  };
+      if (!value && columnId !== 'status.code' && columnId !== 'error_count') return '-';
+
+      switch (columnId) {
+        case 'endTime':
+          return moment(value).format('MM/DD/YYYY HH:mm:ss.SSS');
```

Comment:
nit. May be move format to constants?

Reply:
i remember there was an issue saying trace analytics doesn't conform to time format in advanced settings, might be better to move all date format to it later like this https://github.com/opensearch-project/dashboards-observability/blob/ffeaa7e33a1659b2df877e79265484075ec77dd6/public/components/trace_analytics/components/common/search_bar.tsx#L30